crepecake/src/main/java/com/google/cloud/tools/crepecake/cache/CacheWriter.java

```diff
@@ -70,4 +79,52 @@ public CachedLayer writeLayer(UnwrittenLayer layer) throws IOException {
       return new CachedLayer(layerFile, compressedBlobDescriptor, diffId);
     }
   }
+
+  /**
+   * Writes a {@link ReferenceLayer} with its corresponding compressed layer content BLOB to cache.
+   */
+  public CachedLayer writeLayer(ReferenceLayer layer, Blob layerContent) throws IOException {
+    File layerFile = getLayerFile(layer.getBlobDescriptor().getDigest());
```

Comment:
can a "layer" not get it's own layerFile?

Reply:
A `CachedLayer` will store a reference to the file it refers to. However, neither a `ReferenceLayer` nor a `DigestOnlyLayer` are cached and don't have an associated file. Revised the comment for `getLayerFile` - the uses of `File`/`toFile` will be all removed in #100